Jenkinsfile

```diff
@@ -59,7 +59,7 @@ pipeline {
                     // related to the key alias within the keystore.
                     withEnv(
                         ['P4A_RELEASE_KEYALIAS=upload',
-                         'P4A_RELEASE_KEYALIAS_PASSWD=$P4A_RELEASE_KEYSTORE_PASSWD']
+                         "P4A_RELEASE_KEYALIAS_PASSWD=$P4A_RELEASE_KEYSTORE_PASSWD"]
```

Comment:
Good catch.

Reply:
The "catch" was that the PR failed! From https://ci.endlessos.org/view/Pull%20Requests/job/kolibri-installer-android-pr/1/:

```
Execution failed for task ':signReleaseBundle'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Failed to read key upload from store "****": Get Key failed: Given final block not properly padded. Such issues can arise if a bad key is used during decryption.
...
[INFO]:    ENV:
...
export P4A_RELEASE_KEYALIAS_PASSWD='$P4A_RELEASE_KEYSTORE_PASSWD'
```

I had a feeling it would be wrong, but that's fun with Jenkins - the only way to find out if something is right is just to try it.